helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,94 @@
+fullName: anchor-platform
+namespace: sandbox
+service:
+   containerPort: 8080
+   servicePort: 8080
+   replicas: 1
+   type: NodePort
+   name: sep
+image:
+   repo: stellar
+   name: anchor-platform
+   tag: 0e4caec
+   pullPolicy: Always
+deployment:
+   envFrom:
+   - secretRef:
+       name: apsigningseed
+ingress:
+   labels:
+     app: appy
+     test: label_test
+   annotations:
+     kubernetes.io/ingress.class: "public"
+     cert-manager.io/cluster-issuer: "default"
+     ingress.kubernetes.io/ssl-redirect: "true"
+     ingress.kubernetes.io/browser-xss-filter: "true"
+     ingress.kubernetes.io/frame-deny: "true"
+     ingress.kubernetes.io/content-type-nosniff: "true"
+     ingress.kubernetes.io/hsts-max-age: "31536000"
+     ingress.kubernetes.io/hsts-include-subdomains: "true"
+   tls:
+     host: "reece.sandbox.kube001.services.stellar-ops.com"
```

Comment:
This exposes some SDF internal sauce. It would be good to change this to more generic default and it make it clear that it needs to be configured